zerver/lib/actions.py

```diff
@@ -5826,3 +5826,7 @@ def get_topic_messages(user_profile: UserProfile, stream: Stream,
         message__recipient=stream.recipient
     ).order_by("id")
     return [um.message for um in filter_by_topic_name_via_message(query, topic_name)]
+
+def check_realm_has_non_limited_plan(realm: Realm) -> None:
```

Comment:
I am not sure whether `actions.py` is the right place for this. Maybe we should create a helpers.py file in views?